docs/notebooks/markov-models.ipynb

```diff
@@ -137,7 +137,7 @@
     "The symbol etymology, which usually gets swept under the rug in mathematically-oriented papers, are as follows: \n",
```

Comment:
<p>Can you clarify this part: "because of the autoregressive component suppressing variability of the emissions" ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ericmjl/bayesian-analysis-recipes/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='104'/>

Reply:
<p>I don't know what I was thinking, and you correctly pointed out something wrong here.</p>